vitals/BatteryConstants.java

```diff
@@ -0,0 +1,38 @@
+package vitals;
+
+public class BatteryConstants {
+
+	public float value, maxLimit, minimumLimit, lowBreach, highBreach;
+    	public String parameter;
+    
+	static final float MAX_CHARGE_RATE= 0.8f;
+	static final float MIN_CHARGE_RATE= 0.0f;

```

Comment:
looks like god ...monster class

Reply:
Seperated the Language constants and used ResourceBundle for handling it